lib/lamian/sidekiq_middleware.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Lamian::SidekiqMiddleware
+  def call(*)
+    Lamian.run do
+      begin
+        yield
+      rescue
+        Raven.extra_context(lamian_log: Lamian.dump(format: :txt))
```

Comment:
Now i do not like `Raven` call in `SidekiqMiddleware`. Maybe there is a way to pass lamian_log via sidekiq error context?

Reply:
The only reason this middleware exists is to make it work with Raven, that's why we are calling Raven here. I this I will rename it to SidekiqRavenMiddleware.

What is sidekiq error context by the way?

Reply:
> What is sidekiq error context by the way?

Second arg in notifier. I have not check where it comes from yet, will do it